can/interfaces/kvaser/structures.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+
+"""
+Contains Python equivalents of the structures in CANLIB's canstat.h, 
+with some supporting functionality specific to Python.
+
+Copyright (C) 2010 Dynamic Controls
```

Comment:
Unless I'm missing something this isn't required

Reply:
Thank you for the fast response. Noticed that the reference to canstat.h was not correct either.